packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm/DataAccessRequestAccessorsFilesForm.tsx

```diff
@@ -544,40 +546,39 @@ export default function DataAccessRequestAccessorsFilesForm(
                       onClearAttachment={onClearAttachment}
                     />
                   </SynapseErrorBoundary>
-
-                  {isRenewal && <Divider sx={{ my: 4 }} />}
                 </>
               )
             }
 
             {
+              // SWC-7319 - Per GovernanceInnovation, do not show these fields on renewal
               // Publications & Summary of Use
-              isRenewal && (
-                <>
-                  <TextField
-                    id={'publications'}
-                    label={'Publication(s)'}
-                    disabled={submitDataAccessRequestIsPending}
-                    multiline
-                    rows={3}
-                    value={publication}
-                    onChange={(e: ChangeEvent<HTMLTextAreaElement>) =>
-                      setPublication(e.target.value)
-                    }
-                  />
-                  <TextField
-                    id={'summaryOfUse'}
-                    label={'Summary of use'}
-                    value={summaryOfUse}
-                    disabled={submitDataAccessRequestIsPending}
-                    multiline
-                    rows={3}
-                    onChange={(e: ChangeEvent<HTMLTextAreaElement>) =>
-                      setSummaryOfUse(e.target.value)
-                    }
-                  />
-                </>
-              )
+              // isRenewal && (
```

Comment:
Unclear future for these fields

Reply:
Do you think we will have a resolution in the near future? If not, maybe just delete the code